### PR TITLE
docs(pokedex): polish readme

### PR DESCRIPTION
**Pull request by alexcg1**
- docs(pokedex): simplify, update readme
- docs(pokedex): restructure
- docs(pokedex): consistent tense in headings
- docs(pokedex): rm behind the scenes section

### CODE REVIEW

File: pokedex-with-bit/README.md [hunk 1]
```diff
@@ -57,117 +39,63 @@ docker run -p 45678:45678 jinahub/app.example.pokedexwithbit:0.0.1-0.9.20
...
 
-![Flow in Dashboard](.github/.README_images/6d28795b.png?raw=true)
+### Build a Docker Image
 
-</td>
-</tr>
-</table>
+After playing with it for a while, you may want to change the code and rebuild the image. Simply run:
+```bash
+docker build -t jinaai/app.examples.pokedexwithbit .
```
Comment (JoanFM) [inline pokedex-with-bit/README.md:95]: I think this is wrong, u need the -f Dockerfile?

Comment (alexcg1) [reply pokedex-with-bit/README.md:95]: I only polished it up, didn't check all commands. But I've run commands exactly the same as that (except image name) before and they've worked fine. As long as there's a `Dockerfile` in the dir, it'll automatically use it so `-f` is optional

Comment (JoanFM) [reply pokedex-with-bit/README.md:95]: okk

**Comment by theUnkownName** [comment]
I like it, looks clean

**Comment by alexcg1** [comment]
@JoanFM can we get an approval? Then we're good to merge and I can tick it off our list!

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**